tests/functional/Columns.spec.ts

```diff
@@ -45,6 +45,7 @@ describe('Test the column example', () => {
             count++;
         })
 		expect(count).toEqual(36);
-		expect(ifcAPI.GetAndClearErrors(m2).size()).toEqual(1);
```

Comment:
@beachtom i suspect the parsing error was that no IfcProject entity is in the model, can you confirm ?